package.json

```diff
@@ -28,25 +28,28 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bs-fetch": "^0.6.2",
-    "graphql": "^15.3.0",
-    "urql": "1.10.0"
+    "bs-fetch": "^0.6.2"
   },
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.9.6",
     "@glennsl/bs-jest": "^0.5.1",
+    "@reasonml-community/graphql-ppx": "^1.0.1",
     "all-contributors-cli": "^6.14.2",
     "babel-jest": "^26.0.1",
     "bs-platform": "7.3.2",
     "gentype": "^3.22.0",
+    "graphql": "^15.3.0",
     "husky": "^4.2.5",
     "lint-staged": "^10.2.2",
     "npm-run-all": "^4.1.5",
-    "reason-react": "^0.7.0"
+    "reason-react": "^0.7.0",
+    "urql": "~1.10.1"
   },
   "peerDependencies": {
     "@reasonml-community/graphql-ppx": "^1.0.1",
-    "reason-react": "^0.7.0"
+    "graphql": "^15.3.0",
```

Comment:
I've only seen this now; I believe the GraphQL PPX doesn't have a peer deps envy range for GraphQ. Given the package's little changes this will need to probably be as loose as urql's range on GraphQL: `^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0` (although you obviously don't have to go as low)

Reply:
Ah good catch, will update!